tests/common/osutil/test_default.py

```diff
@@ -99,16 +106,15 @@ def test_mount_dvd_failure(self, _):
         with patch.object(osutil.DefaultOSUtil,
                           'get_dvd_device',
                           return_value='/dev/cdrom'):
-            with patch.object(shellutil,
-                              'run_get_output',
-                              return_value=(1, msg)) as patch_run:
+            with patch.object(shellutil, 'run_command',
+                              side_effect=shellutil.CommandError("mount dvd", 1, "", msg)) as patch_run:
                 with patch.object(os, 'makedirs'):
                     try:
                         osutil.DefaultOSUtil().mount_dvd()
                         self.fail('OSUtilError was not raised')
                     except OSUtilError as ose:
                         self.assertTrue(msg in ustr(ose))
-                        self.assertTrue(patch_run.call_count == 6)
+                        self.assertEqual(patch_run.call_count, 5)
```

Comment:
Why did the number of calls change if we didn't change the retry number (6)?

Reply:
The range has always been `[1..6)`, so I'm not quite sure how this test was passing before. Any ideas?

Reply:
Ah, ok I figured it out. Before, we were mocking `run_get_output`, because that's what `self.mount` used before. There is a `run_get_output` call  in `mount_dvd` that we decided not to bother changing (as it uses a constant string), which was the 6th call before. Since our mock is now for `run_command`, that `run_get_output` call obviously doesn't get counted anymore, bringing the total number of calls from 6 (5 `run_get_output` calls in `self.mount` + 1 `run_get_output` in `mount_dvd` itself) to 5 (just the five `run_command` calls in `self.mount`).

Reply:
Thanks for checking! Makes sense now.